src/engine/GraphStoreProtocol.cpp

```diff
@@ -68,13 +69,31 @@ std::vector<SparqlTripleSimpleWithGraph> GraphStoreProtocol::convertTriples(
 ParsedQuery GraphStoreProtocol::transformGet(const GraphOrDefault& graph) {
   // Construct the parsed query from its short equivalent SPARQL Update string.
   // This is easier and also provides e.g. the `_originalString` field.
-  std::string query;
-  if (const auto* iri =
-          std::get_if<ad_utility::triple_component::Iri>(&graph)) {
-    query = absl::StrCat("CONSTRUCT { ?s ?p ?o } WHERE { GRAPH ",
-                         iri->toStringRepresentation(), " { ?s ?p ?o } }");
-  } else {
-    query = "CONSTRUCT { ?s ?p ?o } WHERE { ?s ?p ?o }";
-  }
+  auto query = [&graph]() -> std::string {
+    if (const auto* iri =
+            std::get_if<ad_utility::triple_component::Iri>(&graph)) {
+      return absl::StrCat("CONSTRUCT { ?s ?p ?o } WHERE { GRAPH ",
+                          iri->toStringRepresentation(), " { ?s ?p ?o } }");
+    } else {
+      return "CONSTRUCT { ?s ?p ?o } WHERE { ?s ?p ?o }";
+    }
+  }();
```

Comment:
1. This is purely a cleanup, if there are more of these, split them up.
2. There is an unnecessary copy here, either `stdd:move` or better (Jason Turner style) don't invoke the lambda immediately, but only in `parseQuery(getQuery())`.